1-js/02-first-steps/15-function-basics/4-pow/task.md

```diff
@@ -4,7 +4,7 @@ importance: 4
 
 # Function pow(x,n)
 
-Write a function `pow(x,n)` that returns `x` in power `n`. Or, in other words, multiplies `x` by itself `n` times and returns the result.
+Write a function `pow(x,n)` that returns `x` in power `n`.
```

Comment:
My suggestion:
```suggestion
Write a function `pow(x,n)` that returns `x` to the power of `n`.
```

How about a wikipedia article for more details instead of `Or, in other words, multiplies `x` by itself `n` times and returns the result.` ?

Reply:
> How about a wikipedia article for more details instead of `Or, in other words, multiplies x by itselfn times and returns the result.` ?

I don't think a separate clarification, much less a wikipedia article, is needed in this case. Exponentiation is a simple mathematical operation known to anyone from comprehensive school. We don't resort to wikipedia articles when addition/subtraction operations are mentioned, right? 
In my opinion, it would be better to leave just a description of what needs to be done here:
> Write a function `pow(x,n)` that returns `x` to the power of `n`.